app/src/main/java/ru/netology/fmhandroid/adapter/PatientListAdapter.kt

```diff
@@ -35,7 +36,18 @@ class PatientListAdapter(
         fun bind(patient: Patient) {
             binding.apply {
                 (patient.lastName + " " + patient.firstName + " " + patient.middleName).also { patientName.text = it }
-                patientLocation.text = if (patient.inHospice) "Да" else "Нет"
+                patientLocation.text =
```

Comment:
Не надо случайно ли переименовать `TextView`? Странно, что в `location` кладётся `status`
```suggestion
                patientStatus.text =
```

Reply:
Да, id остался с прошлой версии. Исправил.